apps/api/src/database/repositories/agentscore-repository.ts

```diff
@@ -56,6 +58,12 @@ async function getAllAgentRanksImpl(): Promise<AgentRankInfo[]> {
       .from(agentScore)
       .innerJoin(agents, eq(agentScore.agentId, agents.id));
 
+    if (agentIds) {
```

Comment:
used below so we can properly sort the "pending" leaderboard and show the ranked agents before unranked ones

Reply:
I say we go with this right now, but just a callout, we need to avoid the were in array for agents queries unless we are super sure the max number of ids is < ~30.

Reply:
ah kk good callut https://github.com/recallnet/js-recall/issues/993